offlineimap/imapserver.py

```diff
@@ -198,7 +198,7 @@ def __md5handler(self, response):
         challenge = response.strip()
         self.ui.debug('imap', '__md5handler: got challenge %s'% challenge)
 
-        passwd = self.__getpassword()
+        passwd = self.__getpassword().encode('ascii')
```

Comment:
I don't think that's correct because this restricts the allowed characters in the password. I wonder we should encode to UTF-8 rather than ascii.